### PR TITLE
Main nav updates, adjust breadcrumb

### PR DESCRIPTION
**Pull request by clarkepaul**
Added “overview” as stack landing, added transitions to nested nav
items, slight adjust meant to breadcrumb to look more like a heading.

### CODE REVIEW

**Comment by clarkepaul** [comment]
There are still some bugs that I couldn't figure out - can I talk you through them @halkyon

**Comment by clarkepaul** [comment]
This will require icons which are in the Platform theme repo

File: templates/Includes/StackIcon.ss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mateusz) [inline templates/Includes/StackIcon.ss]: Umm not sure what this is used for? Doesn't seem to be referenced anywhere.

Comment (clarkepaul) [inline templates/Includes/StackIcon.ss]: Ah, yup that wasn't used in the end. Needs to be removed

Comment (halkyon) [inline templates/Includes/StackIcon.ss]: @clarkepaul can you remove it please? :) I don't have control over this PR

File: code/control/DNRoot.php [hunk 2]
```diff
@@ -675,6 +680,7 @@ public function postsnapshotsuccess(SS_HTTPRequest $request) {
 	 * @return \SS_HTTPResponse
 	 */
 	public function project(SS_HTTPRequest $request) {
+		$this->isOverview = true;
```
Comment (mateusz) [inline code/control/DNRoot.php]: Use `$this->setCurrentActionType(DNRoot::PROJECT_OVERVIEW);` and then use that in the ->Menu call?

**Pull request closed without merging**